calicoctl/commands/clientmgr/client.go

```diff
@@ -45,7 +47,11 @@ func NewClient(cf string) (*client.Client, error) {
 // otherwise will load from environment variables.
 func LoadClientConfig(cf string) (*api.CalicoAPIConfig, error) {
 	if _, err := os.Stat(cf); err != nil {
-		log.Infof("Config file cannot be read - reading config from environment")
+		errMsg := fmt.Sprintf("Config file: %s cannot be read - reading config from environment", cf)
+		if (cf != constants.DefaultConfigPath) {
+			fmt.Fprintln(os.Stderr, errMsg)
```

Comment:
I think we shouldn't try to re-use the error message for the failure case and the warning case, since they're really different behaviors.

i.e., right now if if fails to read the file and exists, it still prints "reading config from environment", which is a lie :)